nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -360,7 +361,7 @@ in {
 
       services.phpfpm = {
         phpOptions = phpOptionsExtensions;
-        phpPackage = pkgs.php71;
+        phpPackage = pkgs.php73;
```

Comment:
Well, if this is hard coded, why is `phpPackages` modifyable? Either both or nothing, everything else just doesn't make any sense

Reply:
I agree. I have no idea why `phpPackages` is configurable. Maybe it would be best to just remove the config option.

Reply:
Yes

Reply:
I removed the `phpPackages` configuration option.